docs/source/examples/spot-jobs.rst

```diff
@@ -369,6 +369,12 @@ The above YAML file defines a pipeline with two tasks. The first :code:`name: pi
 
 To submit the pipeline, the same command :code:`sky spot launch` is used. The pipeline will be automatically launched and monitored by SkyPilot. You can check the status of the pipeline with :code:`sky spot queue` or :code:`sky spot dashboard`.
 
+.. note::
+
+  The :code:`$SKYPILOT_TASK_ID` environment variable is also available in the :code:`run` section of each task. It is unique for each task in the pipeline.
+  For the :code:`eval` task, the :code:`$SKYPILOT_TASK_ID` will be:
```

Comment:
One UX discussion: I found myself reading these as "spot_id". Took a while to realize that it should be read as "_spot" and appended with "id-xxxx". Shall we skip the `_id` part if backward compatible?

Reply:
Oh, it should be read as `spot_id` and appended with `_<job_id>-<task_id>`. I guess there will be no significant backward compatibility error in the case. I am not strongly for keeping the `id` or not. Wdyt?

Reply:
Hmm, reason I found "spot_id" odd is that typically we can just append the IDs without saying IDs first. (E.g., we prepend the timestamp without saying `timestamp-`.)

In this case, how about we remove `-id`? I guess moving `spot-id` as a whole may be less desirable.

Reply:
We do have an `_id` for normal cluster job. To keep it align, we may have to remove it for those jobs as well. 
Removed is for both now.